src/components/options/style.scss

```diff
@@ -0,0 +1,34 @@
+.options {
```

Comment:
As before, slap a "this is temporary" comment at the top and I'm happy for this to merge.